backend_new/src/services/listing.service.ts

```diff
@@ -20,6 +26,10 @@ import {
 import { AmiChart } from '../dtos/ami-charts/ami-chart.dto';
 import { ListingViews } from '../enums/listings/view-enum';
 import { TranslationService } from './translation.service';
+import { ListingCreate } from '../dtos/listings/listing-create.dto';
+import { SuccessDTO } from '../dtos/shared/success.dto';
+import { ListingUpdate } from '../dtos/listings/listing-update.dto';
+import { firstValueFrom } from 'rxjs';
```

Comment:
preference: specify imports from libraries before local files

Reply:
I like that! I'll make sure everything else follows this as well!